less/webtorrent.less

```diff
@@ -2,9 +2,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this file,
  * You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-@import "./variables.less";
-@import "./button.less";
-@import "./about/siteDetails.less";
+@import "variables.less";
```

Comment:
Deprecated with https://github.com/brave/browser-laptop/pull/10881/files#diff-9774044dc3cb63ab522b79039669a42aR17 and https://github.com/brave/browser-laptop/pull/10881/files#diff-9774044dc3cb63ab522b79039669a42aR18